docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+services:
+  bot:
+    restart: unless-stopped
+    container_name: bot
+    image: containers.pkg.github.com/Webeleon/discord-bot-starter
+    environment:
+      PORT: 5000
+      DISCORD_API_TOKEN:
+      DISCORD_CLIENT_ID:
+      MONGO_URL:
```

Comment:
It would be awesome if the composed dockers run their own MongoDB.
Without exposing the port to the outside to avoid conflict with local installations.